sig-platform/charter.md

```diff
@@ -1 +1,42 @@
-To be created by SIG...
\ No newline at end of file
+# Charter
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](../sig-charter-guide.md) and uses the Roles and Organization Management outlined in [sig-goverance](../sig-governance.md).
+ 
+## Scope
+
+The Platform SIG maintains the core ODH operator, and establishes the standards and best practices for deployments, ecosystem integration, builds, monitoring, and secruity. 
+
+body of expertise necessary to ensure architectural consistency over time.
+
+## In scope
+
+- Standards for telemetry, integrations, testing, builds, DevOps, and Security
+- Deployments (embedded ODH, edge deployments, airgaped, embedded) scalability
+- ODH Operator
+- ODH Core Component Integration
+- Code, Binaries, Design and Services Covered
+- Opendatahub-io and opendatahub-io-contrib GitHub org
+- Monitoring, metrics
+- Platform Authentication/ Authorization
+- Service Mesh
+
+## Out of scope
+
+- Work covered under DevEx(UI/UX, Dashboard) and MLOps
+
+
+## Members:
+* Chairs: 
+  * Taneem Ibrahim (Red Hat)
+  * Landon LaSmith (Red Hat)
+  * Samu Veloso Lopez (Red Hat)
+  * Gabe Goodheart (IBM)
```

Comment:
We should default to the person's name with no employer and a link to their GitHub profile

Reply:
Updated. The k8s charters typically have company associations, so I followed that.